Adyen/services.py

```diff
@@ -322,6 +329,42 @@ def payment_result(self, request=None, **kwargs):
         action = "paymentsResult"
         return self.client.call_checkout_api(request, action, **kwargs)
 
+    def payments_captures(self, request, idempotency_key=None, path_param=None, **kwargs):
+        if path_param == "":
+            raise ValueError(
+                'must contain a pspReference in the path_param, path_param cannot be empty'
+            )
+        action = "paymentsCapture"
+        return self.client.call_checkout_api(request, action, path_param, idempotency_key, **kwargs)
```

Comment:
From #144 I see that the arg order for the `call_checkout_api` function was flagged, shouldn't path_param go after idempotency_key?

Reply:
I missed it when I fixed the branch.